war/src/main/less/base/style.less

```diff
@@ -422,24 +422,6 @@ pre.console {
   color: gray;
 }
 
-.spinner {
-  padding-left: 32px;
-  padding-top: 0.5em;
-  padding-bottom: 0.5em;
-  background-image: url("../../images/spinner.gif");
-  background-repeat: no-repeat;
-  background-position: left;
-}
-
-.spinner-right {
```

Comment:
I think these should be kept for backwards compatibility. Did you run an search for uses of these classes?

Reply:
I have, looks like the only use cases of them is where plugins have copied `hudson-behaviour.js` so should be fine :) 

The `.gif` itself has been left in for backwards compat as its still used for `select` input loaders, but should be able to replace that with the future forms PR.